### DISCUSSION TITLE
bromite systemwebview page is not accurate

### DISCUSSION THREAD

**Discussion by pepekinho** [bromite/bromite#2002] [General]
I was trying to install the bromite webview on my old phone with lineage os 13 (android 6.0.1)
and I noticed that android 6 doesn't have config_webview_packages.xml and the feature for choosing the webview on the developers option  was introduced on android 7.0 so technically  with the new name (org.bromite.webview) you can't install the bromite webview on android 6.0 only the vanilla chromium webview (com.android.webview) is still an option.
so right now the description is
*Marshmallow 6.0 and above (SDK23)* 
and it should be 
^ Nougat 7.0 and above (SDK24)

**Comment by csagan5** [comment]
Someone managed to do it also on Marshmallow: https://forum.xda-developers.com/t/magisk-mod-oos-9-5-11-13-allow-choosing-bromite-as-webview-implementation.3954582/post-80832671

I will update the pages to mention that SDK24 is necessary for the SystemWebView.

**Reply by csagan5** [reply]
Fixed.
